### DISCUSSION TITLE
How to make assets/*.file path in PublicPath in the d.ts ?

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wxt-dev/wxt#476] [General]
![image](https://github.com/wxt-dev/wxt/assets/21095710/4eadc3b1-3dfa-452c-8f43-5cf2e3815017)

**Comment by aklinker1** [comment]
You can't. "assets" must be imported into JS files, they're transformed by vite and their filenames will be inconsistent in the output directory.

In your case, `injected.css` probably shouldn't be an asset. It should probably be an entrypoint or a public asset - ***move it to either the `entrypoints/` dir or `public/` dir, and the file will show up in the public path type***.

What's the difference for a CSS file?

- `assets/` - Put CSS files here if they are linked to by multiple HTML files, like base stylesheets shared by your entire app
- `public/` - Put CSS files here if they don't need to be processed by vite, they're just copied as-is to the output directory (SCSS files don't go well here)
- `entrypoints/` - If they are injected into a page, they're considered an entrypoint, similar to content scripts. Entrypoints are processed by vite and are output to a consistent file path in the output directory.

**Reply by zouhangwithsweet** [reply]
`injected.css` is a bundle file. Not a static file.

**Reply by aklinker1** [reply]
Then put it in `entrypoints/` and refer to the [CSS entrypoint docs](https://wxt.dev/entrypoints/css.html). Or, if it is related to a content script, see the [content script CSS docs](https://wxt.dev/entrypoints/content-scripts.html#css)
